packages/dds/test-dds-utils/package.json

```diff
@@ -33,11 +67,13 @@
 		"prettier:fix": "prettier --write . --cache --ignore-path ../../../.prettierignore",
 		"test": "npm run test:mocha",
 		"test:coverage": "c8 npm test",
-		"test:mocha": "npm run test:mocha:base -- \"dist/test/**/*.spec.*js\"",
+		"test:mocha": "npm run test:mocha:esm && echo skipping cjs to avoid overhead - npm run test:mocha:cjs",
```

Comment:
nit: skip cjs testing

Reply:
As a test util used in other test, probably should test both.